src/status_im/translations/ar.cljs

```diff
@@ -29,11 +29,11 @@
    :suggestions-requests                  "الطلبات"
    :suggestions-commands                  "الأوامر"
 
-   ;sync
+                                        ;sync
```

Comment:
@yenda Can we please not due format changes in PRs in future? It makes it hard to review the actual diff.

Reply:
ok I moved the reformatting in #3204